api/functions/getLocation/getLocation.js

```diff
@@ -0,0 +1,17 @@
+import db from "../../db.js";
+
+export const getLocation = async (crsCode) => {
+	const result = await db.query(
+		"SELECT latitude, longitude FROM uk_stations WHERE crs_code = $1",
+		[crsCode],
+	);
+	const location = {
+		location: {
+			latLng: { ...result.rows[0] },
+		},
+	};
+
+	return result.rowCount != 0
+		? location
```

Comment:
What's the contract for what's returned from this function? On success you're returning { latitude:float, longitude:float} and on fail you are returning { message : string}.  As a consumer of the function, how do I know it's worked? Checking for existence of a field is.... meh.  Maybe return { success: boolean, lat: float, lng:float, message:string } ???  Think as a consumer: what do you want to do?

Reply:
return object updated with proper success and message properties